hdx_hapi/endpoints/models/admin_level.py

```diff
@@ -2,51 +2,60 @@
 from typing import Optional
 from pydantic import ConfigDict, Field
 from hdx_hapi.endpoints.models.base import HapiBaseModel
+from hdx_hapi.config.doc_snippets import (
+    DOC_LOCATION_NAME,
+    DOC_LOCATION_CODE,
+    DOC_ADMIN1_CODE,
+    DOC_ADMIN1_NAME,
+    DOC_ADMIN2_NAME,
+    DOC_ADMIN2_CODE,
+    DOC_REFERENCE_PERIOD_START,
+    DOC_REFERENCE_PERIOD_END,
+    truncate_query_description,
+)
 
 
 class LocationResponse(HapiBaseModel):
     id: int
-    code: str = Field(max_length=128)
-    name: str = Field(max_length=512)
-    from_cods: bool
-    reference_period_start: datetime
-    reference_period_end: Optional[datetime]
+    code: str = Field(max_length=128, description=truncate_query_description(DOC_LOCATION_NAME))
+    name: str = Field(max_length=512, description=truncate_query_description(DOC_LOCATION_CODE))
+
+    reference_period_start: datetime = Field(description=DOC_REFERENCE_PERIOD_START)
+    reference_period_end: Optional[datetime] = Field(description=DOC_REFERENCE_PERIOD_END)
 
     model_config = ConfigDict(from_attributes=True)
 
 
 class Admin1Response(HapiBaseModel):
     id: int
     location_ref: int
-    code: str = Field(max_length=128)
-    name: str = Field(max_length=512)
-    from_cods: bool
```

Comment:
from_cods needs to be in response

Reply:
@danmihaila I've fixed the failing tests and also updated text following @turnerm's suggestions